src/steps/stepsContent.js

```diff
@@ -18,7 +18,7 @@ export const stepsContent = {
       exterior: [
         { value: "roof", text: 'Roof' },
         { value: "windows", text: 'Windows' },
-        { value: "exteriorPaint", text: 'Exterior Paint' },
+        { value: "exterior-paint", text: 'Exterior Paint' },
```

Comment:
If we change it from `'exterior-paint'` to `exteriorPaint` than we need to change it here also.